ibis/backends/dask/tests/conftest.py

```diff
@@ -19,6 +19,8 @@ def npartitions():
 
 
 class TestConf(PandasTest):
+    supports_structs = False
```

Comment:
I see that you added the test table below, but the structs are not supported?

Reply:
Removed the test table.